core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -61,7 +61,8 @@ public interface Configs {
 
   ConfigEntry<String> ENTRY_KV_ROCKSDB_BACKEND_PATH =
       new ConfigBuilder(ENTITY_KV_ROCKSDB_BACKEND_PATH_KEY)
-          .doc("Directory path of `RocksDBKvBackend`")
+          .doc(
+              "The storage path for RocksDB storage implementation. It supports both absolute and relative path, if the value is a relative path, the final path is `${GRAVITINO_HOME}/${PATH_YOU_HAVA_SET}`, default value is `${GRAVITINO_HOME}/data/rocksdb`")
```

Comment:
The line is too long, we typically have 100 letters limitation.

Reply:
Okay, let me simplify it.